report_qweb_signer/README.rst

```diff
@@ -0,0 +1,111 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+==================
+PDF reports signer
+==================
+
+This module extends the functionality of report addon to sign
+PDFs using a PKCS#12 certificate.
+
+
+Installation
+============
+
+To install this module, you need to install Java JDK::
+
+  apt-get install openjdk-7-jre-headless
+
+
+Configuration
+=============
+
+In order to start signing PDF documents you need to configure certificate(s)
+to use in your company.
+
+* Go to ``Settings > Companies > Companies > Your company``
+* Go to ``Report configuration`` tab
+* Click ``Edit``
+* Add a new item in ``PDF report certificates`` list
+* Click ``Create``
+* Set name, certificate file, password file and model
+* Optionally you can set a domain and filename pattern for saving as attachment
+
+For example, if you want to sign only customer invoices in open or paid state:
+
+* Model: ``account.invoice``
+* Domain: ``[('type','=','out_invoice'), ('state', 'in', ('open', 'paid'))]``
+* Save as attachment: ``(object.number or '').replace('/','_') + '.signed.pdf'``
+
+**Note**: Linux user that executes Odoo server process must have read access
+to certificate file and password file
+
+
+Usage
+=====
+
+User just prints PDF documents (only Qweb PDF reports supported) as usual,
+but signed PDF is automatically downloaded if this document model is configured
+as indicated above.
+
+If 'Save as attachment' is configured, signed PDF is saved as attachment and
```

Comment:
But I don't think this would happen any time. If you want to store the documents, you want them signed, not unsigned.